generators/authentication/templates/config/passport.js

```diff
@@ -152,6 +152,11 @@ module.exports = {
     onPassportAuth(req, res, error, user, info) {
       if (error || !user) return res.negotiate(error || info);
 
+      if (info.name === 'TokenExpiredError') {
+        info.status = 401
+        return res.negotiate(info)
```

Comment:
@mikedevita yeah, I think we need `return` here, since we don't want `res.ok` to be called. I'd like to write this code in one-line style. Something like `if (info.name === 'TokenExpiredError') return res.negotiate(info);`.